staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -66,6 +67,9 @@ type AuditLogOptions struct {
 	MaxBackups int
 	MaxSize    int
 	Format     string
+	// Preferred group version for the log output.
```

Comment:
Same comment as in https://github.com/kubernetes/kubernetes/pull/51719#discussion_r136378611, word "preferred" sounds little bit confusing

Reply:
Same in webhook options and in flag descriptions